app/misc/repo.ts

```diff
@@ -2,7 +2,7 @@ let Git = require("nodegit");
 let repoFullPath;
 let repoLocalPath;
 let bname = {};
-let branchCommit = [];
```

Comment:
branchCommit was completely unused and was within the code we worked on, so I took the liberty to remove it